src/rules/style.ts

```diff
@@ -47,6 +53,10 @@ export default {
   'one-variable-per-declaration': true,
   // Requires that import statements be alphabetized.
   'ordered-imports': false,
+  // Warns for class methods that do not use ‘this’. <FEEDBACK>
+  'prefer-function-over-method': false,
+  // Prefer foo(): void over foo: () => void in interfaces and types. <FEEDBACK>
+  'prefer-method-signature': true,
```

Comment:
8 errors:
```
app/components/RichTextEditor/RichTextEditor.tsx[7, 3]: Use a method signature instead of a property signature of function type.
app/components/Sidebar/Actions.tsx[2, 3]: Use a method signature instead of a property signature of function type.
app/components/Sidebar/Item.tsx[18, 3]: Use a method signature instead of a property signature of function type.
app/components/Sidebar/Sidebar.tsx[24, 3]: Use a method signature instead of a property signature of function type.
app/components/Sidebar/Sidebar.tsx[43, 3]: Use a method signature instead of a property signature of function type.
app/components/TitleBar/TitleBar.tsx[12, 16]: Use a method signature instead of a property signature of function type.
packages/@shopify/quilt/components/ChoiceList/ChoiceList.tsx[21, 3]: Use a method signature instead of a property signature of function type.
packages/@shopify/quilt/components/EventListener/EventListener.tsx[6, 3]: Use a method signature instead of a property signature of function type.
```